src/helpers/getOriginalContractName.ts

```diff
@@ -0,0 +1,19 @@
+import { goerliProvider, mainnetProvider } from '@/helpers/providers'
+import Badge from '@/models/Badge'
+import Network from '@/models/Network'
+import data from '@/data'
+import getContract from '@/helpers/getContract'
+
+export default function (badge: Badge) {
+  if (badge.type === 'Email') {
+    return badge.original
+  } else {
```

Comment:
You don't need `else` here

Reply:
`if (...) { return } else { return }` equals to: 
```TS
if (...) return 

return
```

Reply:
you're right, thank you, fixed